sycl/test-e2e/Matrix/SG32/get_coord_int8_matB.cpp

```diff
@@ -11,8 +11,6 @@
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 
-// XFAIL: *
```

Comment:
Same here:
@Nuullll, can you please comment on whether we should keep xfail::cpu here?
"get_coord" was also corrected on the CPU for packed B, right?

Reply:
"XFAIL: cpu" should be kept here.